packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -1,3 +1,146 @@
+        ___ 
+        **Tip**: Test code between a pair of `startTest` and `stopTest` methods can run synchronously. To debug your asynchronous functionality, use these methods within your tests.  
+        ___
+    * Batch, Queueable, and Scheduled Apex
+    * Inbound email
+    * Code with the `@future` annotation  
+
+* Keep these things in mind when working with breakpoints.
+    * You can’t set conditional breakpoints.
+    * Breakpoints set on a get or set method must be within the method’s body.
+    * You can’t set breakpoints in or step through Execute Anonymous blocks. However, when you hit a breakpoint using Execute Anonymous, we show your Execute Anonymous frame in the stack. To view your Execute Anonymous code’s variables, click this line in the stack.  
+
+* Keep these things in mind when working with variables.
+    * You can’t watch variables.
+    * Variable inspection in dynamic Visualforce and Lightning components isn’t supported.
+    * You can’t drill into the instance variables of Apex library objects. To view these objects’ contents, use their `toString` methods.
+    * Variables declared within a loop are visible outside of the loop.
+    * Drill into variables to see their children’s values. For example, if you run the query `[SELECT Id, ContactId, Contact.accountId, Contact.Account.ownerId FROM Case]`, your results are nested as follows.
+        ```
+        Case
+        --> Contact
+        -----> contactId
+        -----> Account
+        --------> accountId
+        --------> ownerId
+        ```
+    * When you perform a SOQL query for variables from the EntityDefinition table, your results include the `durableId` even if you don’t explicitly `SELECT` that variable.
+
+## Resources
+* Visual Studio Code Docs: [Debugging](https://code.visualstudio.com/docs/editor/debugging)
+* Trailhead: [Get Started with Salesforce DX](https://trailhead.salesforce.com/trails/sfdx_get_started)
+* _[Salesforce DX Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup)_
+* _[Apex Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode)_
+* _Force.com IDE Developer Guide_: [Explore a Simple Debugging Puzzle](https://developer.salesforce.com/docs/atlas.en-us.eclipse.meta/eclipse/debugger_puzzle_parent.htm)
+* GitHub: [salesforcedx-vscode-apex-debugger](https://github.com/forcedotcom/salesforcedx-vscode-apex-debugger)
+
+---
+Currently, Visual Studio Code extensions are not signed or verified on the Microsoft Visual Studio Code Marketplace. Salesforce provides the Secure Hash Algorithm (SHA) of each extension that we publish. Please consult [Manually Verify the salesforcedx-vscode Extensions’ Authenticity](https://developer.salesforce.com/media/vscode/SHA256.md) to learn how to verify the extensions.  
```

Comment:
@DatGuyJonathan - Could you check that we do generate the SHA1 for the debugger and we do insert it into the SHA256.md file?

Reply:
I believe so, the command is in the [package.json](https://github.com/forcedotcom/salesforcedx-vscode/blob/release/v41.3.0/packages/salesforcedx-vscode-apex-debugger/package.json#L45) and I see the SHA in the doc.